crates/turbopack-core/src/reference_type.rs

```diff
@@ -60,6 +58,8 @@ pub enum CssReferenceSubType {
 #[derive(Debug, Clone, PartialOrd, Ord, Hash)]
 pub enum UrlReferenceSubType {
     EcmaScriptNewUrl,
+    CssAtImport,
```

Comment:
I'm not a fan of this being a `URL` reference type. `@import` and `composes` should be CSS reference types.

Could we just move the common logic from `url_resolve` instead a shared method between `url_resolve` and `css_resolve` so that we are not forced to use an `URL` reference type.

(Note: There are some special rules applied to `URL` reference types, which shouldn't affect CSS references)